pkgs/development/libraries/quickfix/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, fetchpatch, autoconf, automake, libtool }:
+
+stdenv.mkDerivation rec {
+  pname = "quickfix";
+  version = "1.15.1";
+
+  nativeBuildInputs = [ autoconf automake libtool ];
```

Comment:
Normally build inputs are placed after src. Also would it be easier to use autoreconfHook here?